tests/rule_based_profiler/alpha/test_profiler_user_workflows.py

```diff
@@ -306,3 +307,99 @@ def test_bobster_profiler_user_workflow_multi_batch_row_count_range_rule_bootstr
             "test_configuration_bootstrap_sampling_method"
         ]["expect_table_row_count_to_be_between_max_value_mean_value"]
     )
+
+
+@pytest.mark.skipif(
+    version.parse(np.version.version) < version.parse("1.21.0"),
+    reason="requires numpy version 1.21.0 or newer",
+)
+def test_quentin_profiler_user_workflow_multi_batch_quantiles_value_ranges_rule(
+    quentin_columnar_table_multi_batch_data_context,
+    quentin_columnar_table_multi_batch,
```

Comment:
If we're type hinting the body, can we also type hint the fixtures?

Reply:
Not necessarily -- we have had a pattern of type hints in the body of a test method, but without type hinting the fixtures.  We need to firm up a standard for this and just follow it as a convention.  Thanks. /cc @donaldheppner